server/app.py

```diff
@@ -117,6 +118,11 @@ def server_error_handler(error):
 
     app.sentry = sentry
     sentry.init_app(app)
+
+    # instantiate registered provider classes (leave non-classes intact)
+    for key, provider in providers.items():
+        providers[key] = provider() if isinstance(provider, type) else provider
```

Comment:
Is this indeed a good place for instantiating all registered ingest classes, i.e. when the application object is created?